Fix clean and clobber targets for fltk and chplvis

tomyhoi opened the pull request.

gbtitus commented on tools/chplvis/Makefile:
```diff
@@ -56,7 +53,9 @@ OFILES= $(SRCS:.cxx=.o) $(GENSRCS:.cxx=.o)
 chplvis: $(GENSRCS) $(OFILES) 
-	$(CXX) -o chplvis $(OFILES) $(LDFLAGS) $(LIBS)
+	$$($(FLTK_CONFIG) --cxx)  -o chplvis $(OFILES) \
+	$$($(FLTK_CONFIG) --ldflags) $$($(FLTK_CONFIG) --libs)
```
Think it might be helpful to indent this second line a few spaces, to make it more obvious that it's a continuation of the command starting on the previous line, rather than a new command. (The trailing backslash on the previous line does say this, but someone reading rapidly might not pick that up.)

gbtitus commented:
Looks good to me. I didn't test it, though.

BTW, you can bring in a participant by mentioning a username, such as @Kyle-B .  My doing so just now will accomplish that. And if you want Kyle (for example) to be the primary reviewer you can set him as the Assignee, over on the right in the web browser view.

Kyle-B commented on third-party/fltk/Makefile:
```diff
@@ -32,12 +32,12 @@ $(FLTK_BUILD_DIR):
+	-rm -f build.$(CHPL_HOST_PLATFORM) install.$(CHPL_HOST_PLATFORM)
 
 clobber:
 	-rm -rf $(FLTK_BUILD_DIR) $(FLTK_INSTALL_DIR) *~
```
This isn't related to your changes, but I'm uneasy about the `rm -rf *~`, why is this needed?

Kyle-B commented:
I'd rather not ignore errors in the Makefile, but this change is fine with me for now.

tomyhoi commented:
I've modified the Makefile to not generate an error if the directory isn't there using a shell if.   I removed the *~ from the clean rules.   I noticed clean and cleanall were the same and just had cleanall depend on clean.

Kyle-B commented:
Looks good to me.
